R/chk-raw.R

```diff
@@ -0,0 +1,40 @@
+#' Check Raw
+#'
+#' @description
+#' Checks if raw using
+#'
+#' `is.raw(x)`
+#'
+#' @inheritParams params
+#' @inherit params return
+#'
+#' @family data_type_checkers
+#'
+#' @seealso [is.raw()]
+#' @seealso For more details about the use of this function,
+#' please read the article
+#' \href{https://poissonconsulting.github.io/chk/articles/chk-families.html}{chk families}.
```

Comment:
This will render nicely on pkgdown.

```suggestion
#' `vignette("chk-families")`.
```

Reply:
Opps I missed - I'll add manually - thanks!